rabbitmq/configure.sls

```diff
@@ -0,0 +1,128 @@
+#!pyobjects
+from __future__ import division
+
+import re
+import socket
+import collections
+from bisect import bisect
+
+include('rabbitmq.service')
+
+
+def update(d, u):
+    for k, v in u.items():
+        if isinstance(v, collections.Mapping):
+            r = update(d.get(k, {}), v)
+            d[k] = r
+        elif isinstance(v, list):
+            d[k] = d.get(k, [])
+            d[k] += v
+        else:
+            d[k] = u[k]
+    return d
```

Comment:
Can you add a comment to link back to this for documentation purposes? http://stackoverflow.com/a/3233356/5221054

It took me some time to realize why it was necessary.